astropy/units/si.py

```diff
@@ -29,12 +29,22 @@
 ###########################################################################
 # LENGTH
 
+# We exclude c as a prefix so we can define cm as a derived unit,
+# not a PrefixUnit. That way it can be used in cgs as a base unit.
 def_unit(
     ["m", "meter"],
     namespace=_ns,
     prefixes=True,
+    exclude_prefixes=["c"],
```

Comment:
If we aren't moving `centimeter` to `ccs` can this be 

```suggestion
    exclude_prefixes=["cm"],
```